plugins/woocommerce/src/Admin/Features/Navigation/CoreMenu.php

```diff
@@ -213,7 +213,8 @@ public static function get_items() {
 			);
 		}
 
-		$add_product_mvp = array();
+		$add_product_mvp  = array();
+		$edit_product_mvp = array();
```

Comment:
I'm not sure whether we need to add this menu item. This would duplicate the `Add New (MVP)` item if it were added to the menu.

Reply:
Hmm good catch, for some reason I thought I need this and really thought I added the `$edit_product_mvp` item as part of the list below. But turns out I never did...
This should be good to remove then as it isn't doing anything.

Reply:
Done as part of: https://github.com/woocommerce/woocommerce/pull/34211/commits/3986d9db8db8ee536261f7fa78302474058ad6e8